ThirdParty/DateTools/DateTools.bundle/am.lproj/DateTools.strings

```diff
@@ -0,0 +1,89 @@
+/* No comment provided by engineer. */
```

Comment:
@kylehickinson - How do we plan to keep this dependency updated? What do you think about fetching this dependency using the bootstrap script similar to YubiKit?

Reply:
Given that:

- We use no code from this dependency
- Only copied over a set of localization/string files
- This is for legacy purposes on iOS 12 anyways and will be eventually removed

I don't think we will update this dependency at all